mod_wasm/modules/wasm/mod_wasm.c

```diff
@@ -304,26 +313,37 @@ static int content_handler(request_rec *r)
        * than what is needed, map all responses to a 500 error.
        */
 
-      if (ret != OK && ret != HTTP_OK) {
+      if (ret != OK && ret != HTTP_OK)
+      {
+        if (buffer != 0)
```

Comment:
Buffer will always be `0` with the way you're calling `ap_scan_script_header_err_strs` above and the latest code at https://github.com/apache/httpd/blob/trunk/server/util_script.c#L892

Maybe you meant to declare it as
`char buffer[MAX_STRING_LEN];`

Reply:
Agree! Updated PR!